desmod/simulation.py

```diff
@@ -140,7 +142,20 @@ def __enter__(self):
             os.chdir(self.workspace)
 
     def __exit__(self, *exc):
-        os.chdir(self.prev_dir)
+        os.chdir(self.orig_dir)
+        try:
+            os.chdir(self.config['sim.workspace'])
+            self.sync()
+        finally:
+            os.chdir(self.orig_dir)
+
+    def sync(self):
+        if self.config['sim.workspace.s3_sync']:
```

Comment:
> self.config.setdefault('sim.sync.s3.enable', False)

Cool.  My original approach was to use `sim.workspace.sync.s3.enable`, but I changed my mind because I thought it might match existing patterns better to be able to simply specify `-s s3_sync True` on the snowbird command-line.  I'm happy to be wrong on this one.

One question: do you I should including "workspace" in `sim.workspace.sync.s3.enable` now that the syncing takes place in `_Workspace`, or is that an unnecessary coupling?